joomla/joomla-cms
# Revise Blog Data to Reflect the No. of Links to 2

In the Blog Sample Data, the menu item 'Home' has the # Links set to 2; whereas, in the article 'About your home page' the number of links mentioned is 4 (Four).

Pull Request for Issue # .
None.  

Refer similar PR - https://github.com/joomla/joomla-cms/pull/17926


**### Summary of Changes**
In the Sample Data's article titled "About your home page", the number of links set is mentioned as 4 where as in the menu item 'Home' it is set as 2.


**### Testing Instructions**

(a) Create a new installation of Joomla! with **BLOG** Sample Data 

(b) Log into the Back-end

(c) Open the menu item "Home" (that is in the Menu titled "Main Menu") for Edit and go to the tab "Blog Layout". Notice that the value shown for the option "# Links" is set to 2 (Two).

![blog_sample_data_menu_item_links_showing_2](https://user-images.githubusercontent.com/4947580/30254232-e6413110-96b2-11e7-968e-5fae61b71e51.jpg)

(d) See the new site created in a separate browser tab

(e) Notice that in the article titled "About your home page", the number of links is mentioned as 4.

![blog_sample_data_original_links_shown_as_4](https://user-images.githubusercontent.com/4947580/30254254-34d1d0d2-96b3-11e7-8c5c-1318c18abd84.jpg)

**### Expected result**

In the article titled "About your home page" the number of link must specify the value 2 - NOT 4.

![blog_sample_data_revised_links_shown_as_2](https://user-images.githubusercontent.com/4947580/30254261-4997867e-96b3-11e7-97e1-b60fa411cd6b.jpg)

**### Actual result**

See the step (e) in the section ### Testing Instructions


### Documentation Changes Required
